tests/Support/SupportLazyCollectionTest.php

```diff
@@ -67,23 +67,6 @@ public function testCanCreateCollectionFromClosure()
         ], $data->all());
     }
 
-    public function testCollect()
```

Comment:
Why delete this test?

Reply:
It moved.

Reply:
Not really. You moved code in src. We should test both cases. The tests don't need to know they happen to be running the same code.

Reply:
@GrahamCampbell I'm not sure what you're saying. The test is here:

https://github.com/laravel/framework/blob/fc7b632cc37a7a54d379a843b7fabb050eea97fa/tests/Support/SupportCollectionTest.php#L3911-L3929

Reply:
There's not a copy of the test in the lazy collection though.